packages/frontend/editor-ui/src/views/TemplatesSearchView.vue

```diff
@@ -301,6 +301,11 @@ onMounted(async () => {
 
 	restoreSearchFromRoute();
 
```

Comment:
This view is rendered only if the user is using a custom hosted templates library